src/ado2gh/Program.cs

```diff
@@ -96,6 +96,12 @@ private static void SetContext(ParseResult parseResult)
 
         private static async Task GithubStatusCheck(ServiceProvider sp)
         {
+            if (Environment.GetEnvironmentVariable("GEI_SKIP_STATUS_CHECK")?.ToUpperInvariant() == "TRUE")
```

Comment:
Two suggestions:
1. Would you put the env variables in [EnvironmentVariableProvider](https://github.com/github/gh-gei/blob/bb3c45aa895eb5ce67d94de115288ccee3a22908/src/Octoshift/Services/EnvironmentVariableProvider.cs. It'd be easier to manage all env variables the cli uses. In this case there is no need to register it as a secret of course. 
2. It would probably be a good idea to also support "1" as a valid valid value so the check would be:
```c#
Environment.GetEnvironmentVariable("GEI_SKIP_STATUS_CHECK")?.ToUpperInvariant() is "TRUE" or "1"
```

Reply:
I did consider both of these options, but ended up following the same pattern as this [prior art](https://github.com/github/gh-gei/blob/ddefd7ac385f38fa4f37f57183a19d40bebe1c62/src/Octoshift/Services/OctoLogger.cs#L40) for consistency

Either way works for me tell me which one makes more sense

Reply:
I still think putting everything is the `EnvironmentVariableProvider` makes more sense since it acts like a central place for reading env. variables. Regarding whether to support "1", I'll leave it to you it doesn't make that much difference anyways.

Reply:
Change the code in 48db53cec7434ca6b73426a6cf767814d4cbc096 to use the `EnvironmentVariableProvider`

Added a `GetValue` to get the value and not mask the value, `GetSecret` uses this method as well

Once this is merged will refactor #1154 to use the same pattern as well